### DISCUSSION TITLE
Access Tribe via multiple Logins?

### DISCUSSION THREAD

**Discussion by dougcyporyn** [robertfmurdock/Coupling#36] [Q&A] [answered]
Hi Rob,

It is sometimes necessary for different team (tribe) members to access the coupling app to generate coupling assignments. Passwords must not be shared among team members. It would be best for each person to login with their own credentials to access their tribe and spin for pairs. Is this currently supported? 

Thanks!
Doug

**Comment by robertfmurdock** [comment, marked as answer]
It sort of is already supported! If I’m understanding you correctly.

When you configure a player, you can enter their email address - if they log in with that email address (either via Google login or a Microsoft account), then they’ll have access to the team.

Does that meet your needs?

**Comment by robertfmurdock** [comment]
Sadly I should make this more obvious. I’m toying with adding a “demo”… this seems like a good candidate for a special highlight TBH

**Comment by dougcyporyn** [comment]
Hi Rob thank you so much for the reply. This really helps.
